### ISSUE TITLE
Treasure Island

### ISSUE THREAD

**Issue by zurda** [fac-15/project-ideas#4]
# Treasure Island

**tl;dr**: Getting recommendations for books, films, and series in one platform.

#### Name:

Michal ([@zurda](https://github.com/zurda))

### The problem I'd like to solve

Most content recommendation platforms are siloed, e.g:
 
- you can easily get a film recommendation after searching for a certain film.
- you can get other series recommendation when searching for a series.
- you can get a book recommendation when searching for a book you like.

I'd like us to create a platform that tailors book/ film/ or series recommendation according to the person's taste in all three. My hypothesis is that that our taste in films also says something about our taste in series and books, and vice versa. Having one platform for all three might allow us to give better recommendations and save people time searching multiple platforms. 

A secondary problem I'd like to solve with this platform is reading more prose (I find that I recently read less prose and more technical books).

### Who are the stakeholders?
- **Television & film viewers**, and **book worms** 
- **Content recommendation platforms**

#### IMDB

You can get film and series recommendations on [IMDB](https://www.imdb.com/title/tt2267998/). Searching for a film like `Gone Girl`, for example, will result in a list of similar films. You can search for a series as well, however, the recommendation will be limited to the type of media you searched for.

#### Goodreads 

You can search for a book on platforms like [Goodreads](https://www.goodreads.com/book/show/4588.Extremely_Loud_and_Incredibly_Close?from_search=true). The search result will include a section: `Readers also like` where you can see similar books.
 
#### Reddit 

You can find Reddit posts with recommendations for: 

- [books to read](https://www.reddit.com/r/booksuggestions/)
- [films to watch](https://www.reddit.com/r/MovieSuggestions/)
- [series to watch](https://www.reddit.com/r/NetflixBestOf/comments/8750io/request_whats_a_good_series_to_binge_watch_when/)
